loadtest/tx.go

```diff
@@ -20,7 +20,7 @@ func SendTx(
 	failureExpected bool,
 	loadtestClient LoadTestClient,
 ) func() {
-	(*txBuilder).SetGasLimit(200000)
+	(*txBuilder).SetGasLimit(300000)
```

Comment:
is this enough? sometimes it will say out of gas but then incrementing it by a little bit will be insufficient

Reply:
Should be here, we're adding another 100k and it's saying that its missing 1.5k